sceptre/scipool/config/develop/sc-tag-options.yaml

```diff
@@ -10,8 +10,7 @@ dependencies:
   - "develop/sc-portfolio-s3-basic.yaml"
   - "develop/sc-portfolio-scheduled-jobs.yaml"
 sceptre_user_data:
-  CostCenters: !from_json
-    - !request 'https://mips-api.finops.sageit.org/tags?limit=25&enable_other_code=true'
+  CostCenters: !file https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.6/tags/SageFinancialProgramCodes.json
```

Comment:
could you load the files from github instead of S3?  the link is easier to access.

Reply:
can do